docs/learn/glossary.md

```diff
@@ -182,32 +182,28 @@ A validator's public address beginning with `terravaloper` followed by a string
 
 ## Testnet
 
-A version of the mainnet just for testing. The testnet does not use real coins. You can use the testnet to get familiar with transactions. The current testnet for Terra is [`pisco-1`](../develop/endpoints.md)
+A copy of the mainnet network used for testing. The testnet does not use real tokens. You can use the testnet to get familiar with carrying out transactions on Terra. The current testnet for Terra is [`pisco-1`](../develop/endpoints.md).
 
 ## The Terra ecosystem
 
 A quickly expanding network of decentralized applications built on the Terra protocol.
 
 ## Tombstone
 
-To block a validator from participating in consensus or oracle voting. Tombstoned validators cannot rejoin the active set.
+A tombstoned validator is blocked from participating in consensus or oracle voting and cannot rejoin the active set.
```

Comment:
We should remove "or oracle voting," as there is no more oracle voting by validators.